util/cliutil/earthlydir.go

```diff
@@ -68,3 +68,23 @@ func DetectHomeDir() (homeDir string, sudoUser *user.User, err error) {
 	}
 	return
 }
+
+// IsBootstrapped provides a tentatively correct guess about the state of our bootstrapping.
+func IsBootstrapped() bool {
+	homeDir, _, err := DetectHomeDir()
+	if err != nil {
+		return false
+	}
+
+	earthlyDir := filepath.Join(homeDir, ".earthly")
+	if !fileutil.DirExists(earthlyDir) {
+		return false
+	}
+
+	installID := filepath.Join(homeDir, ".earthly", "install_id")
```

Comment:
Will this heuristic work even if analytics reporting is turned off via config?

Reply:
Probably not. I actually think we should remove this as a check since its not even directly controlled by bootstrap.